cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/OCI_TERMINOLOGY.md

```diff
@@ -0,0 +1,122 @@
+# OCI TERMINOLOGY
```

Comment:
This is great, and I'm really fine with this file being at this place for now, however, for leveraging the website docs rendering, we should move it eventually to `site/content/docs/latest/reference/developer`.